python/lsst/afw/geom/transform/transformContinued.py

```diff
@@ -33,10 +35,20 @@ def getJacobian(self, x):
                     self.getFromEndpoint().getNAxes())
     return matrix
 
+
+def of(self, before):
+    if isinstance(before.getToEndpoint(), type(self.getFromEndpoint())):
```

Comment:
I'm glad equality works.

I'm surprised that flake8 objected to `type(...) == type(...)`. It's perfectly valid, though one rarely should need to do it. For fun I made a small snippet and flake8 had no objections to it:

    a = "string"
    b = "another"
    assert type(b) == type(a)